npm-shrinkwrap.json

```diff
@@ -366,78 +380,78 @@
               "resolved": "https://registry.npmjs.org/libbase64/-/libbase64-0.1.0.tgz"
             },
             "libqp": {
               "version": "0.1.1",
-              "from": "libqp@^0.1.1",
+              "from": "https://registry.npmjs.org/libqp/-/libqp-0.1.1.tgz",
               "resolved": "https://registry.npmjs.org/libqp/-/libqp-0.1.1.tgz"
             }
           }
         },
         "hyperquest": {
           "version": "0.3.0",
-          "from": "hyperquest@^0.3.0",
+          "from": "https://registry.npmjs.org/hyperquest/-/hyperquest-0.3.0.tgz",
           "resolved": "https://registry.npmjs.org/hyperquest/-/hyperquest-0.3.0.tgz",
           "dependencies": {
             "through": {
               "version": "2.2.7",
-              "from": "through@~2.2.0",
+              "from": "https://registry.npmjs.org/through/-/through-2.2.7.tgz",
               "resolved": "https://registry.npmjs.org/through/-/through-2.2.7.tgz"
             },
             "duplexer": {
               "version": "0.1.1",
-              "from": "duplexer@~0.1.0",
+              "from": "https://registry.npmjs.org/duplexer/-/duplexer-0.1.1.tgz",
               "resolved": "https://registry.npmjs.org/duplexer/-/duplexer-0.1.1.tgz"
             }
           }
         },
         "libmime": {
           "version": "0.1.6",
-          "from": "libmime@^0.1.5",
+          "from": "https://registry.npmjs.org/libmime/-/libmime-0.1.6.tgz",
           "resolved": "https://registry.npmjs.org/libmime/-/libmime-0.1.6.tgz",
           "dependencies": {
             "iconv-lite": {
               "version": "0.4.5",
-              "from": "iconv-lite@^0.4.4",
+              "from": "https://registry.npmjs.org/iconv-lite/-/iconv-lite-0.4.5.tgz",
               "resolved": "https://registry.npmjs.org/iconv-lite/-/iconv-lite-0.4.5.tgz"
             },
             "libbase64": {
               "version": "0.1.0",
-              "from": "libbase64@^0.1.0",
+              "from": "https://registry.npmjs.org/libbase64/-/libbase64-0.1.0.tgz",
               "resolved": "https://registry.npmjs.org/libbase64/-/libbase64-0.1.0.tgz"
             },
             "libqp": {
               "version": "0.1.1",
-              "from": "libqp@^0.1.1",
+              "from": "https://registry.npmjs.org/libqp/-/libqp-0.1.1.tgz",
               "resolved": "https://registry.npmjs.org/libqp/-/libqp-0.1.1.tgz"
             }
           }
         },
         "nodemailer-direct-transport": {
           "version": "1.0.0",
-          "from": "nodemailer-direct-transport@^1.0.0",
+          "from": "https://registry.npmjs.org/nodemailer-direct-transport/-/nodemailer-direct-transport-1.0.0.tgz",
           "resolved": "https://registry.npmjs.org/nodemailer-direct-transport/-/nodemailer-direct-transport-1.0.0.tgz",
           "dependencies": {
             "smtp-connection": {
               "version": "0.1.7",
-              "from": "smtp-connection@^0.1.5",
```

Comment:
After a second look at this, `version` and `resolved` haven't changed, so it seems to be ok. NTC.